docs/src/modules/spring/pages/side-effects.adoc

```diff
@@ -1 +1,43 @@
 = Running Side Effects
+
+include::ROOT:partial$include.adoc[]
+
+== Emitting effects on another component
+
+An Entity or an Action may also emit one or more side effects. A side effect is something whose result has no impact on the result of the current command--if it fails, the current command still succeeds. The result of the side effect is therefore ignored. When used from inside an Entity, side effects are only performed after the successful completion of any state actions requested by the command handler.
+
+There is no guarantee that a side effect will be executed successfully. If a failure occurs after the command is fully handled, effects might not be executed. Side effects are not retried in case of failures.
+
+Side effects may be declared as synchronous or asynchronous. Asynchronous commands run in a "fire and forget" fashion. The code flow of the caller (the command handler of the entity which emitted the asynchronous command) continues while the command is being asynchronously processed. Meanwhile, synchronous commands run sequentially, that is, the commands are processed in order, one at a time. The final result of the command handler, either a reply or a forward, is not sent until all synchronous commands are completed.
+
+=== Use case: mobile notification
+
+You might want to run side effects to notify interested parties of a change in state. For example, after a withdrawal is made from a bank account, an account entity could send a notification to the account owner's mobile phone.
+
+
+== Emitting a side effect
+
+To illustrate how you can emit a side effect, you can build on top of the xref:spring:actions-as-controller.adoc[Action as a Controller] example. In that previous example, you  build a controller around the xref:value-entity.adoc[Value Entity Counter] and forwarded the incoming request
+after modifying it.
+
+This time, instead of using a `forward`, you will call the entity using a side effect.
+
+== Implementing the Action
+
+The class `DoubleCounterAction` listens to the counter state changes. When the counter value changes this action doubles doubles it. 
+
+[source,java,indent=0]
+.src/main/java/com/example/actions/DoubleCounterAction.java
+----
+include::example$spring-valueentity-counter/src/main/java/com/example/action/DoubleCounterAction.java[tag=controller-side-effect]
+----
+<1> Retrieving the id of the counter 
+<2> On incoming request, doubling the value of `increase`, 
+<3> Building a reply using `Confirmed.getDefaultInstance()`. And attaching a side effect, i.e. calling to the `Counter` to increase double the previous amount.
+
+Please note that, the result of a side effect is ignored by the command meaning that even if the call to
```

Comment:
I know that this is coming from the original documentation, but maybe better to say that the "result is ignored". As it is now it may give the impression that the side-effect is not even run.

Reply:
totally

Reply:
I think it may be better `the result of a side effect is ignored` for `the response of a side effect is ignored` @Renato I think without the reference to side effect the reader may not be sure of what result are we talking about.

Reply:
@franciscolopezsancho, you tagged another Renato here.